ingestion/src/metadata/ingestion/source/trino.py

```diff
@@ -53,6 +54,16 @@ def get_connection_url(self):
 
 class TrinoSource(SQLSource):
     def __init__(self, config, metadata_config, ctx):
+        try:
+            from pyhive import hive
+        except ModuleNotFoundError:
+            pass
+        if "hive" in locals():
```

Comment:
this should be "pyhive" in locals() right?

Reply:
I don't think so.
```
Python 3.9.7 (default, Nov 18 2021, 01:57:42)
[Clang 13.0.0 (clang-1300.0.29.3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> "hive" in locals()
False
>>> "pyhive" in locals()
False
>>> from pyhive import hive
>>> "pyhive" in locals()
False
>>> "hive" in locals()
True
>>>
```

Reply:
ya that doesn't work in ubunu
![image](https://user-images.githubusercontent.com/38649/146661356-342b5e4e-bb71-4912-9a97-45a2ea580a12.png)

Reply:
an or condition is better
``` 
if "hive" in locals() or "pyhive" in locals:
```
to cover different import methods

Reply:
Fixed.